webpack.config.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
   },
   resolve: {
     alias: {
-      'dist/rx.all.js': 'rx/dist/rx.all.js'
+      'dist/rx.all.js': 'rx/dist/rx.all.js',
+      rx: path.resolve(path.join(__dirname, 'node_modules', 'rx'))
```

Comment:
Isn't this the default lookup mechanism?

Reply:
Yes, but when left to its own devices, webpack pulls in rx from `node_modules/redux-epic/node_modules/rx`, as well as `node_modules/rx`

Because `redux-epic` itself isn't bundled, it brings all of its mates to the party.

Reply:
Looks the package lock for redux-epic hasn't modified the requirements of rx. Seems like the version bump did not normalize the versions.

Reply:
It's strange, it still packs a different version for redux epic even though they both resolve to the same hash

![1__tmux](https://user-images.githubusercontent.com/6775919/35454999-33bd58ea-0285-11e8-8e92-17543f995d99.png)

Reply:
I'll do an `npm remove redux-epic` then reinstall it to see if it helps

Reply:
Running `npm dedupe` fixed it for me